src/Minitwit.Infrastructure/Repository/ReactionRepository.cs

```diff
@@ -8,16 +8,12 @@ public class ReactionRepository(MinitwitDbContext dbContext) : BaseRepository(db
 {
     public async Task AddReaction(ReactionType reaction, Guid cheepId, Guid authorId)
     {
-        Cheep? cheep = await db.Cheeps.FindAsync(cheepId);
-        Author? author = await db.Users.FindAsync(authorId);
-        if (cheep != null)
+        if (cheepId != Guid.Empty && authorId != Guid.Empty)
```

Comment:
Can't imagnie such a scenario. A cheep that doesn't exist, cannot recieve reactions because the only way for a user to react is through the UI, where a non-existing cheep wouldn't appear